lib/model/query/users.js

```diff
@@ -43,6 +43,9 @@ const invalidatePassword = (user) => ({ Sessions, run }) => Promise.all([
   run(sql`update users set password=null where "actorId"=${user.actor.id}`),
   Sessions.terminateByActorId(user.actorId)
 ]);
+invalidatePassword.audit = (user) => (log) => log('user.update', user.actor, {
+  data: { password: null }
+});
```

Comment:
This is similar to `updatePassword.audit` just a few lines above. You can also see in `update.audit` above that when a user's basic details are changed, `user.update` logs the user's new data: the `data` property of the audit log details is the user's new data. `updatePassword.audit` does something similar, logging `{ data: { password: true } }`. It doesn't log the new password of course, but rather just `true` to mean "changed to something". I thought about logging `{ data: { password: false } }` here. However, I think `{ password: null }` makes more sense, as the `password` field actually is changed to `null`, and it's OK to log that. And if `{ password: true }` means "changed", I think `{ password: false }` would mean "not changed".